zephyr/esp32/include/sdkconfig.h

```diff
@@ -1,22 +1,7 @@
-/* TODO:
- * move to Kconfig and remove file
+/*
+ * Copyright (c) 2024 Espressif Systems (Shanghai) Co., Ltd.
+ *
+ * SPDX-License-Identifier: Apache-2.0
  */
-#ifdef CONFIG_MCUBOOT
-#define CONFIG_EFUSE_VIRTUAL_OFFSET 0x250000
-#define CONFIG_EFUSE_VIRTUAL_SIZE 0x2000
-#define CONFIG_EFUSE_MAX_BLK_LEN 192
-#endif
 
 #define BOOTLOADER_BUILD 1
```

Comment:
Would be good to understand the issue when this is removed.

Reply:
- when BOOTLOADER_BUILD is defined the implementation of bootloader_flash_read() calls bootloader_flash_read_allow_decrypt()
- when BOOTLOADER_BUILD is not defined the implementation of bootloader_flash_read() calls esp_flash_read()